src/main/java/org/ohdsi/cohortcharacterization/CCQueryBuilder.java

```diff
@@ -134,27 +136,36 @@ private String renderCustomAnalysisDesign(FeatureAnalysis fa, Integer cohortId)
 		);
 	}
 
-	private List<String> getQueriesForCustomDistributionAnalyses(final Integer cohortId) {
+	private List<String> getQueriesForCustomDistributionAnalyses(final Integer cohortId, CohortCharacterizationStrata strata) {
+
+		Long strataId = Objects.nonNull(strata) ? strata.getId() : 0L;
+		String strataName = Objects.nonNull(strata) ? strata.getName() : "";
 
 		return cohortCharacterization.getFeatureAnalyses()
 						.stream()
 						.filter(fa -> fa.getType().equals(StandardFeatureAnalysisType.CUSTOM_FE))
```

Comment:
to be consistent can you use  `Objects.equals(fa.getType(), StandardFeatureAnalysisType.CUSTOM_FE)` method